library/core/src/ptr/mut_ptr.rs

```diff
@@ -776,7 +779,7 @@ impl<T: ?Sized> *mut T {
     /// of this issue.
     #[unstable(feature = "const_raw_ptr_comparison", issue = "53020")]
     #[rustc_const_unstable(feature = "const_raw_ptr_comparison", issue = "53020")]
-    #[inline]
+    #[inline(always)]
```

Comment:
I've heard that `guaranteed_{ne,eq}` become a ton of MIR. I guess that doesn't matter since LLVM is what does inline(always), so I suppose this can stay...

But I'd rather not eat my words here and these functions are only really ever called in const, so I don't think they're really good candidates.